Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnFocusReceiver.cs

```diff
@@ -25,11 +47,9 @@ public InteractableOnFocusReceiver(UnityEvent ev) : base(ev)
         /// <inheritdoc />
         public override void OnUpdate(InteractableStates state, Interactable source)
         {
-            bool changed = state.CurrentState() != lastState;
-
             bool hasFocus = state.GetState(InteractableStates.InteractableStateEnum.Focus).Value > 0;
 
-            if (hadFocus != hasFocus && changed)
+            if (hadFocus != hasFocus)
```

Comment:
FYI @Troy-Ferrell interactable doesn't actually update its state when instantiated from code -- it is always in "disabled" state. Troy are you already fixing this or should I file a new bug?

Reply:
Note: I still think this change is valid, since i'm not sure why we need to predicate this particular event on a state change. We are already tracking state via hadFocus.

Reply:
@julenka, no my changes atm are currently all around theme consumption/management in Interactable

Reply:
Creatd #5866 5866